lib/elixir/lib/enum.ex

```diff
@@ -1126,13 +1126,13 @@ defmodule Enum do
 
   @doc """
   Invokes the given function to each item in the enumerable to reduce
-  it to a sigle element, while keeping an accumulator.
+  it to a single element, while keeping an accumulator.
 
   Returns a tuple where the first element is the mapped enumerable and
   the second one is the final accumulator.
 
-  The function `fun` receives two arguments, the first one is the
-  element, and the second one is the accumulator; and it must return a
+  The function, `fun`, receives two arguments: the first one is the
```

Comment:
Perhaps 'fun' shouldn't be preceded by "The function..." as well.

On Fri, Oct 16, 2015 at 4:35 PM eksperimental <notifications@github.com>
wrote:

> In lib/elixir/lib/enum.ex
> <https://github.com/elixir-lang/elixir/pull/3876#discussion_r42300842>:
>
> >
> >    Returns a tuple where the first element is the mapped enumerable and
> >    the second one is the final accumulator.
> >
> > -  The function `fun` receives two arguments, the first one is the
> > -  element, and the second one is the accumulator; and it must return a
> > +  The function, `fun`, receives two arguments: the first one is the
>
> I dont think 'fun' should be surrounded by commas here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/3876/files#r42300842>.
>